src/coreComponents/constitutive/unitTests/testCompositionalProperties.cpp

```diff
@@ -96,33 +89,32 @@ class CompositionalPropertiesTestDataTestFixture : public ::testing::TestWithPar
     // Compare against numerical derivatives
     // -- Pressure derivative
     real64 const dp = 1.0e-4 * pressure;
-    computeMolarDensity( pressure-dp, temperature, composition, currentMolarDensity );
-    fdDerivative = -(currentMolarDensity - molarDensity) / dp;
-    checkDerivative( dMolarDensity_dp, fdDerivative, "Molar density left pressure derivative" );
-    computeMolarDensity( pressure+dp, temperature, composition, currentMolarDensity );
-    fdDerivative = (currentMolarDensity - molarDensity) / dp;
-    checkDerivative( dMolarDensity_dp, fdDerivative, "Molar density right pressure derivative" );
+    internal::testNumericalDerivative(
+      pressure, dp, dMolarDensity_dp,
+      [this, & t=temperature, & zmf=composition]( real64 const p ) -> real64 {
+      return computeMolarDensity( p, t, zmf );
+    } );
+
     // -- Temperature derivative
     real64 const dt = 1.0e-6 * temperature;
```

Comment:
very minor, but can we take advantage of this PR to note `dT` rather than `dt` 
```suggestion
    real64 const dT = 1.0e-6 * temperature;
```